src/main/java/com/MyApp/budgetControl/domain/expense/category/CategoryJpaRepository.java

```diff
@@ -0,0 +1,11 @@
+package com.MyApp.budgetControl.domain.expense.category;
+
+import java.util.List;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+public interface CategoryJpaRepository extends CategoryRepository, JpaRepository<CategoryEntity, Long> {
```

Comment:
Please remove public, we want to encapsulate repositories inside the package and they should not be accessible from outside of the package - we should only make CategoryService public to make sure that no one will use repository directly instead of using service. 

The purpose for this is that service may protect some rules around entities that will not be applied if You use repository directly

Reply:
In general every time when You are making some class/method public think twice if it really needs to be public and why

Reply:
Changed access modifiers of clases to default (package level) only service and DTOs classes left with the public modifier, due to use outside "category" pacakge